src/plugins/walletConnectToDapps/eventsManager/useWalletConnectEventsHandler.ts

```
@@ -92,11 +98,45 @@ export const useWalletConnectEventsHandler = (
 
         case EIP155_SigningMethod.WALLET_ADD_ETHEREUM_CHAIN:
         case EIP155_SigningMethod.WALLET_SWITCH_ETHEREUM_CHAIN:
-          // This doesn't fit our usual pattern and doesn't require a modal
-          // We implicitly return a happy JSON-RPC response
+          const rpcParams = params.request.params as WalletSwitchEthereumChainParams
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)